scripts/foglamp

```diff
@@ -163,7 +163,35 @@ foglamp_reset() {
 
   # Remove user data: scripts
   rm -rf "$FOGLAMP_DATA/scripts"
-  echo "Removed user data in $FOGLAMP_DATA/scripts"
+  echo "Removed user data from $FOGLAMP_DATA/scripts"
+  # Remove user data: logs
+  find "$FOGLAMP_DATA/logs" -depth -type f -exec rm {} \;
```

Comment:
@pintomax will this remove directories under logs as well?

Reply:
It currently removes files, directories under log are kept

Reply:
We only need to preserve, what foglamp have when it starts. Let me find the directory tree structure,  We see, when (vanilla) foglamp starts.

Reply:
If we need to remove child directories I'll add a change

Reply:
pi@raspberrypi:/usr/local/foglamp/data $ tree
	.
	├── core.err
	├── etc
	│   ├── certs
	│   │   ├── admin.cert
	│   │   ├── admin.csr
	│   │   ├── admin.key
	│   │   ├── ca.cert
	│   │   ├── ca.key
	│   │   ├── foglamp.cert
	│   │   ├── foglamp.csr
	│   │   ├── foglamp.key
	│   │   ├── json
	│   │   ├── pem
	│   │   ├── user.cert
	│   │   ├── user.csr
	│   │   └── user.key
	│   ├── kerberos
	│   │   └── README.rst
	│   └── storage.json
	├── extras
	│   └── fogbench
	│       └── fogbench_sensor_coap.template.json
	├── foglamp.db
	├── foglamp.db-shm
	├── foglamp.db-wal
	├── logs
	└── var
	    └── run
	        └── foglamp.core.pid

	10 directories, 19 files


@pintomax  
1) We need to keep `pem` and `json` directories under `$FOGLAMP_DATA/etc/certs`
2) remove everthing under `$FOGLAMP_DATA/logs` so just rm -rf $FOGLAMP_DATA/logs/*